.gitlab-ci.yml

```diff
@@ -919,6 +919,10 @@ deploy_deb:
     - ls $OMNIBUS_PACKAGE_DIR
   tags: [ "runner:main", "size:large" ]
   script:
+    # We first check that the current version hasn't already been deployed
+    # (same as the check_already_deployed_version). We do this twice to mitigate
+    # races and issues with retries while failing early if there is an issue.
+    - /deploy_scripts/fail_deb_is_pkg_already_exists.sh
```

Comment:
does the script work when the working dir isn't `$OMNIBUS_PACKAGE_DIR`?

Reply:
You're right, fixed that